### DISCUSSION TITLE
Rating value has minimum value as 0 which needs to be removed.

### DISCUSSION THREAD

**Discussion by techframewirk** [beckn/protocol-specifications#118] [Core Specification]
Currently in the core specification the value property in Rating schema object has the minimum value as 0. The core specification of the protocol should not have this limitation and the high and low value should be flexible as per any use cases that a network may have.

**Comment by ravi-prakash-v** [comment]
You are right @techframewirk. The minimum and maximum value should be a BPP policy. Having said that, how will a BAP be aware of this rating policy?
